conf/application.conf

```diff
@@ -151,3 +151,8 @@ play.filters.cors {
   allowedHttpMethods = ["POST", "OPTIONS"]
   allowedOrigins = ["https://www.tax.service.gov.uk"]
 }
+
+help-frontend {
+  host = "http://localhost:9240"
```

Comment:
Should this be added to the `app-config-env` files?

Reply:
On an environment it will not use this config, this is needed for local development only.